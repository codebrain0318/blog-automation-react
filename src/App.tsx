import React, { Suspense } from 'react';
import './App.css';
import Spinner from './components/Spinner/Spinner';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import { GlobalProvider } from './components/GlobalContext';

const DefaultLayout = React.lazy(() => import("./layout/DefaultLayout"));

function App() {
  return (
    <BrowserRouter>
      <GlobalProvider>
        <Suspense fallback={<Spinner />}>
          <Routes>
            <Route path="*" id="Home" element={<DefaultLayout />} />
          </Routes>
        </Suspense>
      </GlobalProvider>
    </BrowserRouter>
  );
}

export default App;
