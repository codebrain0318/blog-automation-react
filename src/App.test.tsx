import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
  it('render snapshot of app', () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});