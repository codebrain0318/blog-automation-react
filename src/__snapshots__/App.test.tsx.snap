// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render snapshot of app 1`] = `
<div
  class="MuiBox-root css-1jo0jhm"
>
  <span
    class="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate css-eglki6-MuiLinearProgress-root"
    role="progressbar"
  >
    <span
      class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate css-5ir5xx-MuiLinearProgress-bar1"
    />
    <span
      class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar2Indeterminate css-1r8wrcl-MuiLinearProgress-bar2"
    />
  </span>
</div>
`;
